docs/configuration.md

```diff
@@ -272,3 +273,41 @@ You shall read output like the following:
 OK
 ```
 If the file is valid YAML but does not contain a valid configuration, it prints a helpful output.
+
+
+## Database backend
+
+Aeternity nodes support several types of database persistence backends:
+ - RocksDB (default for Unix, available for Unix)
```

Comment:
Could you please add newline here as well, it's required in the standard markdown. Github preview unfortunately support only their own frankenstein.

However, docs.aeternity.com uses the standard markdown parser.